### ISSUE TITLE
Not support uninstall from pip

### ISSUE THREAD

**Issue by goooxu** [Microsoft/nni#102]
Followed the below commands to install nni
```
pip3 install -v --user git+https://github.com/Microsoft/nni.git@v0.1
source ~/.bashrc
```

then try to uninstall it by
```
pip3 uninstall NNI
```

after uninstalled, try to run `nnictl`, got the below message
![image](https://user-images.githubusercontent.com/22703054/45795498-91c9b380-bcce-11e8-8e85-fb82e64b88f2.png)

but expect
![image](https://user-images.githubusercontent.com/22703054/45795307-b3766b00-bccd-11e8-8238-5089d8859954.png)

**Comment by QuanluZhang** [comment]
could you help fix this bug?

**Comment by scarlett2018** [comment]
had released in v0.3
